packages/dds/tree/src/feature-libraries/chunked-forest/chunkTree.ts

```diff
@@ -244,7 +248,9 @@ export function tryShapeFromSchema(
 	return getOrCreate(shapes, type, () => {
 		const treeSchema = schema.nodeSchema.get(type) ?? fail("missing schema");
 		if (treeSchema instanceof LeafNodeStoredSchema) {
-			return new TreeShape(type, true, []);
+			return treeSchema.leafValue === ValueSchema.String
```

Comment:
This code makes an interesting and possibly suppressing choice, which while correct, might seem like a bug or might want to be changed later, so we should probably document it:
```suggestion
			// Allow all string values (but only string values) to be compressed by the id compressor.
			// This allows compressing all compressible identifiers without requiring additional context to know which values could be identifiers.
			// Attempting to compress other string shouldn't have significant overhead,
			// and if any of them do end up compressing, that's a benefit not a bug.
			return treeSchema.leafValue === ValueSchema.String
```

Reply:
Added documentation clarifying for why we compress both identifiers and string values which happens to be compressible identifiers